spec/rubocop/cop/lint/unused_block_argument_spec.rb

```diff
@@ -143,6 +143,18 @@
       end
     end
 
+    context 'when a keyword argument is not used' do
+      let(:source) { <<-END }
+        1.times do |foo:|
+          puts 'foo'
```

Comment:
Something like `define_method` is the only reasonable use of keyword arguments with blocks I can think of, so it would be a better example here.